quipucords/scanner/network/inspect_callback.py

```diff
@@ -43,6 +44,7 @@ def __init__(self, scan_task, inspect_results, display=None):
         self.results = []
         self._ansible_facts = {}
 
+    @transaction.atomic
```

Comment:
This shouldn't be needed now.  I think having _finalize_host atomic is enough.

Reply:
Removed, thank you!